components/org.wso2.carbon.identity.multi.attribute.login/org.wso2.carbon.identity.multi.attribute.login.resolver.regex/src/main/java/org/wso2/carbon/identity/multi/attribute/login/resolver/regex/RegexResolver.java

```diff
@@ -106,6 +124,9 @@ public AuthenticationResult authenticateWithIdentifier(String loginAttributeValu
             UserRealm userRealm = UserResolverUtil.getUserRealm(tenantDomain);
             UniqueIDUserStoreManager userStoreManager = UserResolverUtil.getUserStoreManager(tenantDomain);
             claimManager = userRealm.getClaimManager();
+            if (allowedAttributes == null) {
+                return authenticationResult;
+            }
```

Comment:
same as above

Reply:
Fixed in [26ef5c0](https://github.com/wso2-extensions/identity-governance/pull/580/commits/26ef5c032a3c5eb7a68ca846b5872b0563d0d793)